lambda/health_package/components/generic_helper.py

```diff
@@ -29,6 +29,7 @@ def get_namespace_service(cls, namespace):
             "AWS/Lambda": "lambda",
             "AWS/Firehose": "firehose",
             "AWS/Kinesis": "kinesis",
+            "aws.codepipeline": "codepipeline",
```

Comment:
I think if AWS aren't going to bother we should force these to be consistent and add a line in to format the namespace value into a consistent pattern. Maybe the easiest thing is just lowercase and then replace / with . which will make them all look like `aws.codepipeline`.  

Otherwise you have to figure out whether to Title Case or UPPERCASE the service name which doesn't seem worth it.